js/src/components/modals/RedirectUpsell.js

```diff
@@ -0,0 +1,178 @@
+import React from "react";
+import styled from "styled-components";
+import interpolateComponents from "interpolate-components";
+import PropTypes from "prop-types";
+import { __, sprintf } from "@wordpress/i18n";
+import { getRtlStyle, utils, YoastModal } from "yoast-components";
+import YoastSeoIcon from "yoast-components/composites/basic/YoastSeoIcon";
+import Icon from "yoast-components/composites/Plugin/Shared/components/Icon";
+
+import UpsellBox from "../UpsellBox";
+
+const { makeOutboundLink } = utils;
+const PremiumLandingPageLink = makeOutboundLink();
+
+const StyledContainer = styled.div`
```

Comment:
This is a literal copy-paste from components/contentAnalysis/SeoAnalysis.js. Please move this to a separate file and include it in both places. Maybe even move it to yoast-components.